.github/workflows/main.yaml

```
@@ -47,11 +47,11 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [18.x, 20.x]
```

Linting only needs to run once using a recent node version:
```suggestion
        node-version: [20.x]
```